### ISSUE TITLE
A11y_.NET Online REPL_Learn_Enter focus mode_ScreenReader: Screen reader is unnecessarily announcing " int int" at last.

### ISSUE THREAD

**Issue by poojapatil36** [dotnet/try#1040]
“[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

### GitHubTags:
#A11yMAS; #A11yTCS;  #A11ySev3; #.NET Online REPL ; #BM-.NET Online REPL-Web-NOV2022; #DesktopWeb; #ScreenReader; #Narrator; #WCAG1.3.1; #FTP; #Win11

### Environment Details:
Application Name: .NET Online REPL
Windows Version: Windows 11
Chromium Edge: Version 108.0.1462.10 (Official build) dev (64-bit)

### Repro Steps:

1. Launch the url https://learn.microsoft.com/en-us/dotnet/csharp/tour-of-csharp/tutorials/hello-world?tutorial-step=1&WT.mc_id=-blog-scottha.
2. Tab to "Enter focus mode" button and hit ENTER key.
3. TAB to .NET editor and type "int".
4. List of suggestion will appear.
5. Observe that screen reader is unnecessarily announcing " int int" at last.

### Actual:
 Screen reader is unnecessarily announcing "int int" at last.

**NOTE:** Issue is reproducing only with Narrator and not with NVDA.

### Expected Result:
Screen reader should not announce "int int" at last unnecessarily.

### Attachment:
![A11y_ NET Online REPL_Learn_Enter focus mode_ScreenReader](https://user-images.githubusercontent.com/97668775/200497325-19083585-2278-492f-8f81-c996f060fca9.gif)
